app/src/interfaces/select-multiple-dropdown/select-multiple-dropdown.vue

```diff
@@ -39,14 +42,14 @@ function updateValue(value: string[]) {
 </script>
 
 <template>
-	<v-notice v-if="!choices" type="warning">
+	<v-notice v-if="!items" type="warning">
```

Comment:
Right. There is unreachable.
I think dropdown type could remove this notice because popup dropdown is already show no options hint. 
Same dispaly logic on `select-dropdown` interface now.